images/kcp-registrar/bin/register.sh

```diff
@@ -214,6 +219,10 @@ register() {
                 --syncer-image "ghcr.io/kcp-dev/kcp/syncer:$KCP_SYNC_TAG" \
                 --resources deployments.apps,services,ingresses.networking.k8s.io,pipelines.tekton.dev,pipelineruns.tekton.dev,tasks.tekton.dev,runs.tekton.dev,networkpolicies.networking.k8s.io \
                 --output-file "$syncer_manifest"
+            # Set a restricted security context
+            patch="$SCRIPT_DIR/../data/syncer-patch.yml" yq -i \
```

Comment:
sometimes, the CI won't play nice with '..'. Please use "$(dirname "$SCRIPT_DIR")")